ui/app/src/modules/Content/Delete/DeleteDialog.tsx

```diff
@@ -272,17 +275,19 @@ function DeleteDialogWrapper(props: DeleteDialogProps) {
   });
 
   useEffect(() => {
-    fetchDeleteDependencies(siteId, selectedItems).subscribe(
-      (response: any) => {
-        setDeleteDependencies({
-          childItems: response.items.childItems,
-          dependentItems: response.items.dependentItems
-        });
-      },
-      (error) => {
-        setApiState({ error });
-      }
-    );
+    if (selectedItems.length) {
+      fetchDeleteDependencies(siteId, selectedItems).subscribe(
+        (response: any) => {
```

Comment:
Remove `any`. Type the service appropriately and the type will just fall through here